### DISCUSSION TITLE
Is it important that the order of migration folders?

### DISCUSSION THREAD

**Discussion by Jihye2ee** [prisma/prisma#6161] [Q&A] [answered]
Hi! I wonder if the order of the migration folder is important as the title. When I was first created, migration was applied with the folder name with time. I removed the time mark without thinking to use folder name easily. and then you run the 'migrate dev' command later in development, it will cause the error as below, so I looked for the reason, and I think the order of the folders is the reason.

![theorder](https://user-images.githubusercontent.com/15521676/111587723-f31e5f00-8805-11eb-9af6-886526850b1c.PNG)
Error: P3006

Migration `2021_02_24_migration` failed to apply cleanly to a temporary database. 
Error code: P1014
Error:
The underlying table for model `nodules` does not exist.

Since it has already been applied to the production version as migrate deploy, you may not be able to rename or delete the folder. How should we solve this?

Will there be a problem if I revise the folder name according to the execution order and upload it to repo and apply it back to production?

Or should I just leave the migration as it is not a problem and develop it with the folder name modified only in the development?

Thanks you in advance.

**Comment by ryands17** [comment, marked as answer]
@Jihye2ee 👋 

If you change the migration name in your folder, you also need to change it in the Migration table in your database. Then it will work fine. This has to be done whenever you create a new migration.

I would suggest keeping the folder names as is with the timestamp though.

**Reply by Jihye2ee** [reply]
Is there no problem updating only the data in the `migration_name ` column with the folder name that I changed?

**Reply by ryands17** [reply]
No just renaming the column in the table and the folder will work fine, but you would need to do it manually :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
